docs/ajaxApi.rst

```diff
@@ -395,7 +395,7 @@ structure of those jobs.
 |                                   |      "type": "job.type"           |
 |                                   |      "in": ["job.ids that this job|
 |                                   |      is directly depending upon.  |
-|                                   |      Indirect ancestors is not in |
+|                                   |      Indirect ancestors are not in |
```

Comment:
This change added 1 more character. The "in" of included should go to next line.